frontends/web/src/routes/settings/manage-accounts.module.css

```diff
@@ -21,18 +21,24 @@
 .acccountLink {
     align-items: center;
     display: flex;
+    flex-basis: calc(100% - 140px);
     flex-grow: 1;
 }
 
 .editBtn {
     background: transparent;
     border: none;
     color: var(--color-primary);
+    flex-basis: 60px;
```

Comment:
quick question, is this intended? The spacing between "Edit" and the toggle is not the same anymore now (bigger).

Before (master):
![Screenshot 2023-07-17 at 17 33 17](https://github.com/digitalbitbox/bitbox-wallet-app/assets/28676406/39fcd235-2af8-4003-97a0-0a004c29e294)


After:
![Screenshot 2023-07-17 at 17 32 45](https://github.com/digitalbitbox/bitbox-wallet-app/assets/28676406/508802f9-26e6-4589-8bc5-d4869ee425c5)

On mobile (>=375px width), without setting the basis still looks fine:
![Screenshot 2023-07-17 at 17 38 05](https://github.com/digitalbitbox/bitbox-wallet-app/assets/28676406/29f6a247-74a3-477d-a709-caad661f28b2)


Do we need to set flex-basis here? 🤔

Reply:
I did change it to flex wrap so the tokens could break into its own section

```
┌────────────────────────────────┬──────┬──────┐
│                                │      │      │
├────────────────────────────────┴──────┴──────┤
│                                              │
│                                              │
│                                              │
│                                              │
└──────────────────────────────────────────────┘
```

but now I noticed that with different translations the text "Edit" in German "Bearbeiten" breaks :( this could be solved by increasing the flex-basis 60px but doesn't feel right.. i.e. with different os the font-rendering can take more space..

I'll try to make the edit column growing...

Reply:
I pushed a fix, please test in mainnet in German and use long account name so that the account name breaks onto 2 lines. I upped to 180px safety, as iirc on Linux font rendering may use more space.